tee-worker/app-libs/stf/src/getter.rs

```diff
@@ -57,7 +61,7 @@ impl From<TrustedGetterSigned> for Getter {
 #[allow(non_camel_case_types)]
 pub enum PublicGetter {
 	some_value,
-	nonce(AccountId),
+	nonce(ShardIdentifier, AccountId),
```

Comment:
Maybe we can reuse `ShardIdentifier` from here:
https://github.com/litentry/litentry-parachain/blob/b940c0a879945dd7d460c2590b59537b2d48a442/tee-worker/core-primitives/stf-executor/src/getter_executor.rs#L56